### ISSUE TITLE
How to train the object detection model for negative samples

### ISSUE THREAD

**Issue by RuchaTambe** [google/automl#695]
Hi, 
I am trying to train an EfficientDet-D0 on a custom dataset which contains images without any bounding box annotations (negative samples in the training dataset). 
For example, the csv file contains  <image_path>,,,,,
where the x-coordinate, y-coordinate of the center of the bounding box, height of bounding box, width of bounding box and label is absent. How can I train a custom dataset with negative samples using EfficientDet-D0?

**Comment by kartik4949** [comment]
Hi, @RuchaTambe 
so if im not wrong, your dataset have some negative examples and some positive examples right?

**Comment by mingxingtan** [comment]
If label is absent, you can set the class id as -1  (meaning ignoring label) in your ground truth.

**Comment by ece85** [comment]
For those that need a few more words of description to have confidence in Mingxingtan answer, i did an experiment to verify that this works for negative training.  I trained on 1 image with one tag to overfit a model.  I then used the same image with 100 images that contained only negative examples to train another model.  I then inferred on a dataset that had mostly empty scenes with both models.  The model trained on with one positive and 100 negative cases had far few false detections.  Therefore, using a classid of -1 and using a bounding box of the entire image,  resulted in behavior that i expected.  thanks @mingxingtan!

**Comment by jimmy821022** [comment]
how to set class id as -1?
